static/app/views/replays/detail/errorList/useErrorFilters.tsx

```diff
@@ -18,28 +18,30 @@ export type FilterFields = {
 };
 
 type Options = {
-  errorCrumbs: Crumb[];
+  errorFrames: ErrorFrame[];
 };
 
 type Return = {
   getProjectOptions: () => ErrorSelectOption[];
-  items: Crumb[];
+  items: ErrorFrame[];
   searchTerm: string;
   selectValue: string[];
   setFilters: (val: ErrorSelectOption[]) => void;
   setSearchTerm: (searchTerm: string) => void;
 };
 
 const FILTERS = {
-  project: (item: Crumb, projects: string[]) =>
+  project: (item: ErrorFrame, projects: string[]) =>
     // @ts-expect-error
     projects.length === 0 || projects.includes(item.data.project || ''),
 
-  searchTerm: (item: Crumb, searchTerm: string) =>
-    JSON.stringify([item.message, item.description]).toLowerCase().includes(searchTerm),
+  searchTerm: (item: ErrorFrame, searchTerm: string) =>
+    [item.message, ...item.data.labels].some(str =>
+      str.toLowerCase().includes(searchTerm)
+    ),
```

Comment:
changed to remove the JSON.stringify business, instead look at `message` and `labels` on their own